denis.roman/task2.rb

```diff
@@ -0,0 +1,2 @@
+hash = { 'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014 }
+p hash.count { |key, value| key.to_s.strip.start_with?('yes') }
```

Comment:
Why would you strip the string before checking if that string starts with 'yes'? whitespaces should be taken into account. Also your code will be case sensitive, that's not a blocker for the merge, but it would be better if your code worked for any case.

Reply:
@RomanDenis

Reply:
@Flippylolz i misread requirement in slack and did the opposite. I ll fix soon